data/human/news.txt

```diff
@@ -1413,7 +1450,7 @@ news "tourist"
 			"Can you tell me again which planet this is? No, I'm not lost, I just don't know where I am."
 			"We did a ride on an ancient spacecraft the other day. It was called something like Soyuz or Shuttle Dragon. I don't remember the name exactly, but it was horrible; my back still hurts from the bumpy ride."
 			"I almost missed my transport last week; someone was wrong on the hypernet and I was in a long, heated discussion to correct it."
-			"Hey! Where's platform 5? I need to board there in a few minutes!"
+			"Hey! Where's Platform ${platform number}? I need to board there in a few minutes!"
```

Comment:
```suggestion
			"Hey! Where's platform ${platform number}? I need to board there in a few minutes!"
```
See line 1834 - looks like platform isn't capitalized there, might as well make it consistent.